sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -155,18 +165,42 @@ export class InstitutionController extends BaseController {
     await this.institutionService.syncInstitution(token);
   }
 
+  /**
+   * Controller method to get all institution users with the
+   * given institutionId institution Admin.
```

Comment:
'for institution Admin' ?

Reply:
only `institution Admin` can view mange users  and manage location